src/python/test/test_dxpy.py

```diff
@@ -1877,6 +1877,23 @@ def test_retry_after_without_header_set(self):
         self.assertTrue(50000 <= time_elapsed)
         self.assertTrue(time_elapsed <= 70000)
 
+    @unittest.skip("Example of a request that should be retried")
```

Comment:
I assume this is under a skip clause because it takes too long? I wonder if we could instrument the code and tests to control the sleep delay between retries and assert on the number of retries made.